applications/app/views/fragments/indexBody.scala.html

```diff
@@ -28,8 +28,14 @@
                 }
 
                 @defining(IndexPage.makeFront(index, Edition(request)).containers) { containers =>
-                    @containers.map { containerDefinition =>
-                        @container(containerDefinition, frontProperties = FrontProperties.fromBranding(branding))
+                    @defining(( containers filterNot (_.isThrasher) map (_.index) ).min ) { containerIndexForBadge =>
+                        @containers.map { containerDefinition =>
+
+                            @container( containerDefinition,
+                                        frontProperties = FrontProperties.fromBranding(branding),
+                                        showFrontBranding =  containerDefinition.index == containerIndexForBadge)
```

Comment:
Could the evaluation of `containerIndexForBadge` be done in a support def elsewhere?
`Defining` blocks in templates are best avoided unless the value is used multiple times.
So something like `def shouldShowFrontBranding(index, containers): Boolean`.